teachertool/src/components/EvalResultDisplay.tsx

```diff
@@ -27,13 +27,17 @@ const ResultsHeader: React.FC = () => {
     );
 };
 
-export const EvalResultDisplay: React.FC<{}> = () => {
+interface EvalResultDisplayProps {
+    printRef: React.RefObject<HTMLDivElement>;
```

Comment:
The name `printRef` seems overly specific here. The ref is just a generic ref, could be used for anything.

Reply:
That's true, it can be used for anything, but I think it's nice to have the specificity here. You can look at the code and know that the ref that is being defined will be used for printing.

Reply:
There's nothing enforcing this usage. It's just a ref. What about naming it for what it points to, maybe `resultsRef`?

Reply:
So, would it be preferred that `printRef` gets renamed to `resultsRef` everywhere?

Reply:
Yes, I think that would work fine.